### PR TITLE
Kill VCM migration UI

### PR DESCRIPTION
**Pull request by orangejenny**
Not quite ready to release this, but opening the PR now so I don't forget to do it once everyone is migrated.

@dannyroberts / @emord

### CODE REVIEW

File: corehq/apps/hqadmin/models.py [hunk 1]
```diff
@@ -81,15 +81,3 @@ def create_pillow_checkpoint_snapshots(cls):
             cls.objects.create(seq=db_seq,
                                checkpoint_id=checkpoint.checkpoint_id,
                                date_updated=date.today())
-
-
-class VCMMigration(models.Model):
```
Comment (emord) [inline corehq/apps/hqadmin/models.py]: Might be nice to have a migration to get rid of this model

File: corehq/reports.py [hunk 2]
```diff
@@ -61,7 +61,6 @@
 def REPORTS(project):
     from corehq.apps.reports.standard.cases.basic import CaseListReport
     from corehq.apps.reports.standard.cases.careplan import make_careplan_reports
-    from corehq.apps.reports.standard.maps import DemoMapReport, DemoMapReport2, DemoMapCaseList
```
Comment (emord) [inline corehq/reports.py]: unrelated change?

Comment (orangejenny) [inline corehq/reports.py]: oops, wrong PR. thanks.

File: corehq/apps/hqadmin/models.py (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/hqadmin/models.py]: https://github.com/dimagi/commcare-hq/pull/13924/commits/9d30b14402190537a191022df1a1f5891241c392

**Review by emord — APPROVED** [review]

**Comment by dannyroberts** [comment]
I'm loving all the red

**Comment by orangejenny** [comment]
@emord i think we're good to release this, don't see a use for the UI anymore
